### DISCUSSION TITLE
Can Goland support Copilot in the future?

### DISCUSSION THREAD

**Discussion by MrNocole** [community/community#11113] [Copilot Conversations] [answered]
I found that go is supported in vscode,but Goland seems not supported.only IDEA and Pycharm is supported(jetbrains IDEs)

**Comment by jansorg** [comment, marked as answer]
@MrNocole GoLand is supported. If there's outdated documentation, we'd be glad for the link.

See https://github.com/github/copilot-docs/blob/main/docs/jetbrains/gettingstarted.md for instructions.
Please make sure you're using 2021.2 or later.
If you can't solve this, please post the info shown by `Help > About`.

**Reply by MrNocole** [reply]
thanks for your help!I successfully installed copilot in GoLand!
